chart/kubeapps/templates/ingress.yaml

```diff
@@ -1,38 +1,38 @@
 {{- if .Values.ingress.enabled -}}
-{{- $fullName := include "kubeapps.fullname" . -}}
-{{- $ingressPath := .Values.ingress.path -}}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ $fullName }}
+  name: {{ template "kubeapps.fullname" $ }}
   labels:
-    app: {{ include "kubeapps.name" . }}
-    chart: {{ include "kubeapps.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-{{- with .Values.ingress.annotations }}
+    app: {{ template "kubeapps.name" $ }}
```

Comment:
Thanks for reverting back to template since it seems that here was the only place in which we used `include`.

Reply:
Why are you using `$` instead of current scope `.`? It seems that in the other templates we use `.`

Reply:
We can use `.`. I'll update it. The `$` was necessary when we were using `{{- range ...` at the top.